source/features/limit-commit-title-length.tsx

```diff
@@ -11,9 +11,14 @@ function init(): void {
 	for (const inputField of inputs) {
 		const validityCallback = (event: Event): void => {
 			if (inputField.value.length > 72) {
-				inputField.setCustomValidity(`The title should be maximum 72 characters, but is ${inputField.value.length}`);
 				event.preventDefault();
 				event.stopImmediatePropagation();
+
+				inputField.setCustomValidity(`The title should be maximum 72 characters, but is ${inputField.value.length}`);
+
+				if (event.type === 'submit') {
+					inputField.reportValidity();
```

Comment:
This still fails to stop "Merge" button from being clicked only on "auto-filled values" (like when editing text-field and then the retained value on reloading the page). If the user had edited text and then clicked merge, this works as intended.

`reportValidity()` is place behind `submit` event check as doing this for each input event looks like constant annoyance.

What if we trigger an `input` event on the fields on page load!?

Reply:
67ea2fcc20b42767ed8c34ed103247727ed29943 should fix this!